src/api/projects/index.js

```diff
@@ -154,3 +155,30 @@ export async function fetchChartData(projectId, days, timezoneOffset) {
     timezoneOffset,
   })).project.chartData;
 }
+
+/**
+ * Fetch releases for a given project
+ *
+ * @param {string} projectId - Project id to fetch releases for
+ * @returns {Promise<Array>} Array of release objects
+ */
+export async function fetchReleases(projectId) {
+  if (!projectId) {
+    throw new Error('projectId is required to fetch releases');
+  }
+  
+  try {
+    const response = await api.call(QUERY_GET_RELEASES, { projectId }, undefined, { allowErrors: true });
+    
+    if (response.errors) {
+      console.error('GraphQL errors:', response.errors);
+      return [];
```

Comment:
GraphQL can return errors along with data (for example, when it can't resolve some field, but have resolved others), so you don't need to return empty array if data is present